packages/ember-cli-code-coverage/lib/gjs-gts-istanbul-ignore-template-plugin.js

```diff
@@ -0,0 +1,47 @@
+const gjsGtsTemplateIgnoreVisitor = {
```

Comment:
@vstefanovic97 let's provide explanation why we add this, which problem it solves and which other options e have considered but didn't go and why.

Reply:
@SergeAstapov I added an explanation to the file